src/category_theory/path_category.lean

```diff
@@ -41,6 +41,25 @@ def of : prefunctor V (paths V) :=
 { obj := λ X, X,
   map := λ X Y f, f.to_path, }
 
+local attribute [ext] functor.ext
+
+/-- Two functors out of a path category are equal when they agree on singleton paths. -/
+@[ext]
+lemma ext_functor {C} [category C]
+  {F G : paths V ⥤ C}
+  (h_obj : F.obj = G.obj)
+  (h : ∀ a b (e : a ⟶ b), F.map e.to_path =
```

Comment:
```suggestion
  (h : ∀ (a b : V) (e : a ⟶ b), F.map e.to_path =
```
Just for readability, to avoid the possible interpretation `(a b : paths V)`.

Reply:
Good point! Before this, Lean actually had `a b : paths V` (but correctly inferred that `e` came from `quiver V`).